jobs/component_jobs.groovy

```diff
@@ -121,11 +121,18 @@ repos.each { Map repo ->
           '''.stripIndent().trim()
         }
         // do not run e2e tests for workflow-manager at this time
-        if (repo.name != 'workflow-manager') {
+        // also don't run if author is 'Jenkins User' as this is from a ci
+        // increment commit which would have already been tested
+        if (repo.name != 'workflow-manager' || '${GIT_COMMITTER_NAME}' != 'Jenkins User' ) {
           downstreamParameterized {
-            trigger("${defaults.testJob[config.type]}") {
+            trigger(downstreamJobName) {
               parameters {
-                predefinedProp(repo.commitEnvVar, '${GIT_COMMIT}')
+                predefinedProps([
```

Comment:
@sgoings 'Aha!' moment right here in figuring out how we can get the test job sending slack notifications to appropriate component channel -- just forward the info to the test job in the form of a param!